package.json

```diff
@@ -2,16 +2,15 @@
   "name": "dash-bootstrap-components",
   "version": "0.2.3",
   "description": "Bootstrap components for Plotly Dash",
-  "main": "lib/index.js",
+  "main": "demo-lib/index.js",
   "scripts": {
-    "build-dist": "npm run clean-lib && npm run extract-metadata && webpack --config=webpack.config.dist.js",
-    "clean-lib": "mkdirp lib && rimraf lib/*",
-    "copy-lib": "copyfiles -u 1 lib/* dash_bootstrap_components",
-    "demo": "webpack-dev-server --hot --inline --port=8888 --content-base=demo --config=webpack.config.demo.js",
-    "extract-metadata": "mkdirp lib && react-docgen --pretty -o lib/metadata.json src/components && node -e \"const fs = require('fs'); const path = require('path'); const m = JSON.parse(fs.readFileSync('./lib/metadata.json')); const r = {}; Object.keys(m).forEach(k => r[k.split(path.sep).join('/')] = m[k]); fs.writeFileSync('./lib/metadata.json', JSON.stringify(r, '\t', 2));\"",
+    "build-dist": "npm run clean:py && npm run build:py && webpack --config=webpack.config.dist.js",
+    "clean:py": "mkdirp dash_bootstrap_components/_components && rimraf dash_bootstrap_components/_components",
+    "demo": "webpack-dev-server --hot --inline --port=8888 --config=webpack.config.demo.js",
+    "build:py": "mkdirp dash_bootstrap_components/_components && dash-generate-components ./src/components dash_bootstrap_components/_components && move-cli dash_bootstrap_components/_components/_imports_.py dash_bootstrap_components/_components/__init__.py",
```

Comment:
Yes, we don't keep the generated sources under version control (unlike, for instance, dash-core-components).

While we could still reformat them with black, I'd vote against because:
- it makes building a little bit longer for no benefit
- it adds more point of failures in the build chain.